odpi.bom

```diff
@@ -132,7 +132,16 @@ bigtop {
                 ref = 'branch-2.7-ODP'; dir = 'branch-2.7-ODP'}
 ***/
     }
-
+    'hive' {
+      name    = 'hive'
+      relNotes = 'Apache Hive'
+      version { base = '1.2.1'; pkg = base+'+odpi'; release = 1 }
+      tarball { destination = "apache-${name}-${version.base}-src.tar.gz"
+                source      = destination }
+      url     { download_path = "/$name/$name-${version.base}/"
+                site = "${apache.APACHE_MIRROR}/${download_path}"
```

Comment:
Ok sounds good. I was using the zookeeper json in the odpi.bom to follow.  Let me look into it!